.github/workflows/ci.yml

```diff
@@ -14,10 +14,9 @@ jobs:
           - 'lts'
           - '1'
           - 'pre'
-          - 'nightly'
         os:
           - ubuntu-latest
-          # - macOS-latest
+          - macOS-latest
           - windows-latest
         arch:
           - x64
```

Comment:
You should probably remove `arch` completely and just rely on the default of setup-julia (uses the architecture of the runner): macOS-latest runs Apple Silicon, so with `x64` you won't run the matching Julia binary.
```suggestion
```